web/src/OverviewTable.tsx

```diff
@@ -277,16 +282,19 @@ function TableEndpointColumn({ row }: CellProps<RowValues>) {
         key={ep.url}
       >
         <StyledLinkSvg />
-        <DetailText>{ep.name || displayURL(ep)}</DetailText>
```

Comment:
hmm...this removes the overflow handling for long urls. was that intentional?

Reply:
Oops! That was a mistake. I did just test it and realized that it wasn't working in the first place because I needed to set a width for the parent. 

Thanks for catching this!